README.md

```diff
@@ -1,27 +1,26 @@
-# blogclub
-this repository is only used for our team to publish blog. if you want to be a member of the team. please see the contact section for help.
```

Comment:
a blog, blogs or the blog?

The noun phrase blog seems to be missing a determiner before it. Consider adding an article.